common/supplemental/subdivisions.xml

```diff
@@ -147,35 +147,34 @@
 		<subgroup type="espv" contains="esbi esss esvi"/>
 		<subgroup type="esri" contains="eslo"/>
 		<subgroup type="esvc" contains="esa escs esv"/>
-		<subgroup type="ET" contains="etaa etaf etam etbe etdd etga etha etor etsn etso etti"/>
+		<subgroup type="ET" contains="etaa etaf etam etbe etdd etga etha etor etsi etsn etso etti"/>
 		<subgroup type="FI" contains="fi01 fi02 fi03 fi04 fi05 fi06 fi07 fi08 fi09 fi10 fi11 fi12 fi13 fi14 fi15 fi16 fi17 fi18 fi19"/>
 		<subgroup type="FJ" contains="fjc fje fjn fjr fjw"/>
 		<subgroup type="fjc" contains="fj09 fj10 fj12 fj13 fj14"/>
 		<subgroup type="fje" contains="fj04 fj05 fj06"/>
 		<subgroup type="fjn" contains="fj02 fj03 fj07"/>
 		<subgroup type="fjw" contains="fj01 fj08 fj11"/>
 		<subgroup type="FM" contains="fmksa fmpni fmtrk fmyap"/>
-		<subgroup type="FR" contains="fr20r frara frbfc frbl frbre frcp frcvl frges frgf frgp frhdf fridf frmf frmq frnaq frnc frnor frocc frpac frpdl frpf frpm frre frtf frwf fryt"/>
+		<subgroup type="FR" contains="fr20r fr971 fr972 fr973 fr974 fr976 frara frbfc frbl frbre frcp frcvl frges frhdf fridf frmf frnaq frnc frnor frocc frpac frpdl frpf frpm frtf frwf"/>
 		<subgroup type="fr20r" contains="fr2a fr2b"/>
-		<subgroup type="frara" contains="fr01 fr03 fr07 fr15 fr26 fr38 fr42 fr43 fr63 fr69 fr73 fr74"/>
+		<subgroup type="frara" contains="fr01 fr03 fr07 fr15 fr26 fr38 fr42 fr43 fr63 fr69 fr69m fr73 fr74"/>
 		<subgroup type="frbfc" contains="fr21 fr25 fr39 fr58 fr70 fr71 fr89 fr90"/>
 		<subgroup type="frbre" contains="fr22 fr29 fr35 fr56"/>
 		<subgroup type="frcvl" contains="fr18 fr28 fr36 fr37 fr41 fr45"/>
-		<subgroup type="frges" contains="fr08 fr10 fr51 fr52 fr54 fr55 fr57 fr67 fr68 fr88"/>
-		<subgroup type="frgf" contains="fr973"/>
-		<subgroup type="frgp" contains="fr971"/>
+		<subgroup type="frges" contains="fr6ae fr08 fr10 fr51 fr52 fr54 fr55 fr57 fr67 fr68 fr88"/>
 		<subgroup type="frhdf" contains="fr02 fr59 fr60 fr62 fr80"/>
-		<subgroup type="fridf" contains="fr75 fr77 fr78 fr91 fr92 fr93 fr94 fr95"/>
-		<subgroup type="frmq" contains="fr972"/>
```

Comment:
@macchiati  it looks like `frmq` became `MQ` so probably need to delete this from other data:

```xml
			<subdivision type="frmq">Martinique</subdivision>
```

Reply:
the subdivision alias was pre-existing, but apparently `frmq` was allowed to "shadow" `MQ` until it was removed as a subdivision alias.

Reply:
I had fixed these in https://github.com/unicode-org/cldr/commit/e6253c6173e1ffe8dd5f5d63a0ee46f4e36ee92d, but it didn't take. (I needed some cleanup in github.) Think it should be good now; we'll see once the tests finish.